CHANGES.rst

```diff
@@ -31,6 +31,12 @@ cube_build
 - Remove trailing dash from IFU cube filenames built from all subchannels.
   Sort subchannels present by inverse alphabetical order to ensure
   consistent filename creation across processing runs. [#6959]
+  
+- Re-wrote c code for NIRSpec dq flagging. In addition,if data is
```

Comment:
I think these are two distinct issues and hence would be nice if they were listed as separate entries in the change log (just so the 2nd one doesn't go unnoticed). It's OK to have to entries reference the same PR number.